22-testing-library/src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <title>State-Testing mit Mocha</title>
```

Comment:
Should this be adjusted to something else, or is this fine?

Reply:
Hi @goloroden: Sorry, this is still a draft - still working on it. Fill fix the comment and assign the PR to you as soon as I am ready. 🙂

Reply:
Aaaaahhh, I missed that this was still a draft 😊